sources/frontend.c

```diff
@@ -1551,6 +1551,24 @@ static inline od_frontend_status_t od_frontend_poll_catchup(od_client_t *client,
 	od_dbg_printf_on_dvl_lvl(
 		1, "client %s polling replica for catchup with timeout %d\n",
 		client->id.id, timeout);
+
+	/*
```

Comment:
Maybe dispatch replication lag between all routes was really poor design-decision. Seems that having some API
for get current last heartbeat (await) would be better.